### PR TITLE
Get newest unique master tag

### PR DESCRIPTION
**Pull request by k2la**
## WHY

This tool is unable to get unique master tag.
## WHAT

It is enable to do that.
I add a function to check unique master tag in tagList.

### CODE REVIEW

File: get.go [hunk 1]
```diff
@@ -14,13 +14,27 @@ import (
 )
 
 func getNewestMasterTag(tagList []string) string {
-
 	for _, tag := range tagList {
-		if strings.Index(tag, "master") != -1 {
-			return tag
+		if strings.Index(tag, "master-") != -1 {
+			if isUnique(tag, tagList) {
```
Comment (koudaiii) [inline get.go]: Why isUnique ? 👀  What's the point of checking there? 👀

File: get.go [hunk 2]
```diff
@@ -14,13 +14,27 @@ import (
...
+				return tag
+			}
 		}
 	}
-	return ""
+	return "master"
```
Comment (koudaiii) [inline get.go]: "latest"

File: get.go (hunk 1 above)
Comment (k2la) [reply get.go]: Take a look at process in isUnique function.

File: get.go (hunk 2 above)
Comment (k2la) [reply get.go]: It was modified.

File: get.go (hunk 1 above)
Comment (dtan4) [reply get.go]: @koudaiii 's comment means that "Is it really necessary to check the uniqueness of `tagList`?"

I think that tags attached to one image are completely unique, so we do not have to check uniqueness again here.

Comment (k2la) [reply get.go]: I understand that.  I will remove isUnique function.

**Comment by k2la** [comment]
I removed isUnique function.

**Comment by k2la** [comment]
Please review 🙇

**Comment by koudaiii** [comment]
LGTM!! Thank you!!

**Pull request merged**